00-intro.md

```diff
@@ -69,8 +69,9 @@ Commands are terse (often only a couple of characters long),
 their names are frequently cryptic,
 and their output is lines of text rather than something visual like a graph.
 On the other hand,
-the shell allows us to combine existing tools in powerful ways with only a few keystrokes
-and to set up pipelines to handle large volumes of data automatically thus improving productivity and reproducibility.
+with only a few keystrokes shell allows us to combine existing tools into 
```

Comment:
"with only a few keystrokes**, the** shell allows us to combine existing tools into..."